Policyfile.lock.json

```diff
@@ -60,25 +48,25 @@
       }
     },
     "chef-client": {
-      "version": "12.3.3",
-      "identifier": "7ab37c81c66070be34d3a5f59cc9c94c87ad00e7",
-      "dotted_decimal_identifier": "34537294493278320.53538329202105545.221330530959591",
-      "cache_key": "chef-client-12.3.3-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/chef-client/versions/12.3.3/download",
+      "version": "12.3.4",
+      "identifier": "9e4148496618fe2253acba978d67921a5dc0b48a",
+      "dotted_decimal_identifier": "44544825045162238.9662150538136935.160641939715210",
+      "cache_key": "chef-client-12.3.4-supermarket.chef.io",
+      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/chef-client/versions/12.3.4/download",
       "source_options": {
-        "artifactserver": "https://supermarket.chef.io:443/api/v1/cookbooks/chef-client/versions/12.3.3/download",
-        "version": "12.3.3"
+        "artifactserver": "https://supermarket.chef.io:443/api/v1/cookbooks/chef-client/versions/12.3.4/download",
+        "version": "12.3.4"
       }
     },
     "cron": {
-      "version": "6.4.0",
-      "identifier": "c112ad00b72022a61c427b3deedef49b3a7e043d",
-      "dotted_decimal_identifier": "54345204755800098.46755917995044574.268947538445373",
-      "cache_key": "cron-6.4.0-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/cron/versions/6.4.0/download",
+      "version": "7.0.4",
```

Comment:
I remember that we had to version-pin this to: https://github.com/lightspeedretail/kitchen-camel/blob/6585d41f197e1507ab44f185338b9902b22b3809/policies/use1prod/_default.rb#L36